content-generation-service/main.tf

```diff
@@ -0,0 +1,154 @@
+locals {
+  name = "content-generation-service"
+}
+
+variable "namespace" {
+  type = string
+}
+
+variable "image_tag" {
+  type = string
+}
+
+variable "image_pull_policy" {
+  type = string
+}
+
+variable "node_pool" {
+  type = string
+}
+
+variable "openai_api_key" {
+  type = string
+}
+
+resource "kubernetes_service" "content-generation-service" {
```

Comment:
I don't see we using the outputs of this kubernetes service in https://github.com/serlo/infrastructure-env-staging/pull/61/files
Are we going to use it in the near future by other microservices in our cluster? 
If not, we could delete this part regading the kubernetes service, since it is for now dead code.

Reply:
Yes, the api.serlo.org will communicate with the content-generation-service.